wardrobe/poe/src/lib.rs

```diff
@@ -85,6 +85,7 @@ pub trait PoeConfig {
 /// waste of caller fees.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, DebugNoBound, CloneNoBound, PartialEq, Eq, TypeInfo)]
+#[scale_info(skip_type_params(T))]
```

Comment:
So what about this change? At some point this helped me beat a compiler error. But now that I've cleaned up the types and traits, it is no longer necessary to build. But I guess maybe it is a good idea anyway?

Reply:
If we dont know why we need it and it doesnt stop us from doing what we are doing Id say best not to have it tbh.

Reply:
Agreed. Removed in 8c09593bd7cbf03e3228dc00f4ff30379ddd6979